### DISCUSSION TITLE
Feature request: Create device for each container, nice and VM

### DISCUSSION THREAD

**Discussion by tim-devel** [dougiteixeira/proxmoxve#315] [Feature request] [answered]
Hi, 

Have just tried your integration for first time and is exactly what I was looking for, thanks for your hard work! 

Can I make a feature request please.  When I first set up the integration it created nearly 200 new entities but no devices.  This makes it really difficult to organise the sensors or group them logically.  Can you create a new device for each node, lxc and VM and then attach the individuals sensor to these devices.

**Comment by dougiteixeira** [comment, marked as answer]
The integration already works with devices for each Proxmox feature.

**Reply by tim-devel** [reply]
It doesn't show them in the integration page.  For most other integrations it will say x devices and y entities.  Screen shot below only lists entities.  

![Screenshot_20240528-091000](https://github.com/dougiteixeira/proxmoxve/assets/6829542/3d4d91bf-a780-453d-933b-b6b16cdc404a)

**Reply by dougiteixeira** [reply]
What we call a device in Home Assistant can be of two types that have different names: Service or Device

The integration already links each entity into a service, which is the same thing as a device, the difference is only in the nomenclature that Home Assistant uses.

We can see from your screenshot that 19 services were created, each service will represent a node, VM, container, disk, storage, etc…
